### PR TITLE
Create Command cheat sheet

### PR DESCRIPTION
**Pull request by mmarchetti**
Create a reference sheet for important Command classes and methods.

### CODE REVIEW

**Comment by mmarchetti** [comment]
See preview [here](https://github.com/Team1559/2024-Crescendo/blob/command-cheat-sheet/Commands.md)

**Comment by BrandonDudek** [comment]
Thanks!

**Comment by BrandonDudek** [comment]
I added some more notes and links, as I was reviewing and researching. Please review and let me know if I got anything wrong.

**Comment by mmarchetti** [comment]
Great additions!

File: Commands.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+cmd.deadlineWith                   |Creates a ParallelDeadlineCommandGroup containing `cmd` and the supplied commands, where `cmd` is the deadline.
+cmd.repeatedly                     |Returns a version of `cmd` that will be run repeatedly, forever.
+cmd.asProxy                        |Returns `cmd` wrapped in a ProxyCommand.    
+cmd.unless                         |Returns a version of `cmd` that will not run if provided boolean supplier returns true.
+cmd.withInterruptBehavior          |Returns a version of `cmd` with the specified interruption behavior. Call with `kCancelIncoming` to make this command non-interruptible.
+cmd.finallyDo                      |Returns a version of `cmd` that runs the provided function when the command ends. The function will receive a boolean interrupted parameter, like the `end` method.
+cmd.handleInterrupt                |Returns a version of this command that will call the provided function when the command is interrupted.
+<br/>**Commands Static Methods**    |<br/>_We generally don't use these; instead, instantiate one of the command classes directly._
+Commands.none                      |Do-nothing command.
```
Comment (BrandonDudek) [inline Commands.md:66]: What is the use case for this?

**Review by BrandonDudek — APPROVED** [review]

**Pull request merged**